### PR TITLE
core/monit: allow theme override for monit widget

### PR DESCRIPTION
**Pull request by fabianfrz**
closes opnsense/plugins#937

### CODE REVIEW

File: src/www/widgets/widgets/monit.widget.php [hunk 1]
```diff
@@ -27,9 +27,15 @@
  */
 
 require_once("widgets/include/monit.inc");
-
+function monitGetJQueryBootGridFile($path) {
```
[the remark this answers is not in the document]
Comment (fabianfrz) [reply src/www/widgets/widgets/monit.widget.php]: @fichtner this is exactly what my function does - it doesn't exist here. Or did I misinterpret your comment. Do you want to tell me that this function should be implemented globally. Am I just confused?

[the remark this answers is not in the document]
Comment (fichtner) [reply src/www/widgets/widgets/monit.widget.php]: this is not a monit-specific function

[the remark this answers is not in the document]
Comment (fichtner) [reply src/www/widgets/widgets/monit.widget.php]: (nor is it a "jquery boot grid" function)

[the remark this answers is not in the document]
Comment (fabianfrz) [reply src/www/widgets/widgets/monit.widget.php]: ok, now I do understand, will fix it now

**Comment by fabianfrz** [comment]
@fichtner thanks for finding it, I've fixed it.

**Comment by fichtner** [comment]
merged manually with minor consistency changes, thanks <3

**Pull request closed without merging**